nilearn/glm/tests/test_first_level.py

```diff
@@ -923,3 +830,258 @@ def test_glm_sample_mask():
               sample_masks=sample_mask)
     assert model.design_matrices_[0].shape[0] == 22
     assert model.predicted[0].shape[-1] == 22
+
+
+"""Test the first level model on BIDS datasets."""
+import os
+
+from pathlib import Path
+
+import pytest
+
+from nilearn._utils.data_gen import create_fake_bids_dataset
+from nilearn.glm.first_level import first_level_from_bids
+from nilearn.interfaces.bids import get_bids_files
+from nibabel.tmpdirs import InTemporaryDirectory
+
+
+def test_first_level_from_bids_bug_3029():
```

Comment:
Is there a convention at nilearn to link a specific regression test to the issue that reported the bug?
should it be referenced at all? Should it be in the name of the test? In the description of the test?

Reply:
I don't think there is. We may have put in the description sometimes. I think that this is good practice.

Reply:
renamed tests and added link to issue in test description